components/System/SelectedDeviceInfo.vue

```diff
@@ -0,0 +1,80 @@
+<template>
```

Comment:
We can also wrap all this component in a transition wrapper so it gets an animation. If you don't think we need here a fast transition, you can use directly https://vuetifyjs.com/en/api/v-fade-transition/ so you don't need to create or import ``assets/transitions.scss``.

I want to move all our own transitions to the vuetify component or create our own, so we can customize them later easily. For now, we can start with this one, so there's one transition less to refactor eventually :):